pkg/epinio/detail/applications.vue

```diff
@@ -51,70 +49,141 @@ export default Vue.extend<Data, any, any, any>({
       } catch (err) {
         console.error(`Failed to scale Application: `, epinioExceptionToErrorsArray(err)); // eslint-disable-line no-console
       }
-
       this.$set(this, 'saving', false);
+    },
+    formatURL(str: string) {
+      const matchGithub = str.match('^(https|git)(:\/\/|@)([^\/:]+)[\/:]([^\/:]+)\/(.+)(.git)*$');
+
+      return `${ matchGithub?.[4] }/${ matchGithub?.[5] }`;
     }
   },
+  computed: {
+    resourcesUssage() {
+      return this.value.instances?.reduce((acc: any, curr: { millicpus: number; memoryBytes: number }) => {
+        acc = {
+          millicpus:   acc.millicpus += curr.millicpus,
+          memoryBytes: acc.memoryBytes += curr.memoryBytes / (1024 * 1024) // bytes to megabytes
+        };
 
-  computed: {}
-
+        return acc;
+      }, {
+        millicpus:    0,
+        memoryBytes: 0,
+      });
+    },
+    choseIcon() {
+      return this.value.metadata.icon || 'fa-cube';
+    },
+    setIconType() {
+      switch (this.value.sourceInfo?.label) {
```

Comment:
Switching on human text generally isn't advisable. The sourceInfo under the hood uses ` switch (this.origin.Kind) { 
    case APPLICATION_MANIFEST_SOURCE_TYPE.PATH:`
...

However, would advise adding an `icon` property to that `sourceInfo` object. To catch the blank case you can just use something like `value.sourceInfo.icon` || `icon-epinio`